mat-process-api/V1/Boundary/PostInitialProcessDocumentRequest.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
+
+namespace mat_process_api.V1.Boundary
+{
+    public class PostInitialProcessDocumentRequest
+    {
+        [Required] public string processRef { get; set; }
```

Comment:
The [Required] ComponentModel DataAnnotations are not needed anymore, because this object's validator takes care of it now.

Reply:
Removed. Reasoning behind the removal is that ... if  for some reason any of these properties would not need to be Required anymore... you'd have to change it in the validations (where you expect) and in the here (where you wouldn't) - it's 2 places doing same thing - not very good anyway.
Also if anyone were to need to add another property, this would cause confusion due to there being 2 places that do this, yet this one being inactive.